packages/react-devtools-shared/src/hook.js

```diff
@@ -473,6 +470,32 @@ export function installHook(target: any): DevToolsHook | null {
     }
   }
 
+  function setStrictMode(rendererID, isStrictMode) {
+    const rendererInterface = rendererInterfaces.get(rendererID);
+    if (rendererInterface != null) {
+      if (isStrictMode) {
+        rendererInterface.patchConsoleForStrictMode();
+      } else {
+        rendererInterface.unpatchConsoleForStrictMode();
+      }
+    } else {
+      // This should only happen during initial render in the extension before DevTools
+      // finishes its handshake with React DOM
```

Comment:
super tiny nit: It's not necessarily React DOM. It's just any injected renderer.